### PR TITLE
Nominatim api

### PR DESCRIPTION
**Pull request by Machou**
Added address markers on LeafLet map

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -10,15 +10,15 @@ import '../styles/app.scss';
...
+//     attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, ' +
+//         'Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+//         id: 'mapbox/streets-v11',
+//     tileSize: 512,
+//     zoomOffset: -1
+// }).addTo(mymap);
 
-var marker = L.marker([49.2257934073045, 4.01539227165092]).addTo(mymap);
\ No newline at end of file
+// var marker = L.marker([49.2257934073045, 4.01539227165092]).addTo(mymap);
```
Comment (rocambille) [inline assets/js/app.js]: dead code: remove it

File: src/Controller/MapController.php [hunk 2]
```diff
@@ -5,16 +5,44 @@
...
+            throw $this->createNotFoundException(
+                'No Fablab found in Falab\'s table.'
+            );
+        }
+
+        $url = 'http://nominatim.openstreetmap.org/';
+        $nominatim = new Nominatim($url);
+
+        $data = [];
+        for ($i = 0; $i < count($fablabRepository); $i++) {
```
Comment (rocambille) [inline src/Controller/MapController.php]: doesn't look like you need a for loop: use a foreach. or even better, give a try to array_map

File: src/Controller/MapController.php [hunk 3]
```diff
@@ -5,16 +5,44 @@
...
+*/
 class MapController extends AbstractController
 {
     /**
-     * @Route("/map", name="map")
+     * @Route("/", name="map")
      */
     public function index(): Response
     {
+        $fablabRepository = $this->getDoctrine()
```
Comment (rocambille) [inline src/Controller/MapController.php]: your variable doesn't contain a repository, but a collection of fablab: something like `$fablabs` would better suit

**Review by rocambille — CHANGES_REQUESTED** [review]

File: assets/js/app.js (hunk 1 above)
Comment (Machou) [reply assets/js/app.js]: done.

File: src/Controller/MapController.php (hunk 2 above)
Comment (Machou) [reply src/Controller/MapController.php]: done.

File: src/Controller/MapController.php (hunk 3 above)
Comment (Machou) [reply src/Controller/MapController.php]: done.

**Pull request merged**